tests/E2E/README.md

```diff
@@ -0,0 +1,50 @@
+# E2E tests
+
+This folder contains tests about ps_banner module. 
+
+## Supported versions
+
+These tests can be run on `1.7.7.x` and `1.7.6.x` PrestaShop versions.
+
+## Stack
+
+In these tests, we used the actual stack :
+
+- [prestashop_test_lib](https://www.npmjs.com/package/prestashop_test_lib) which uses [playwright](https://playwright.dev/) as a node library to automate browsers.
+- [mocha](https://mochajs.org/) as a test framework.
+- [chai](https://www.chaijs.com/) as an assertion library.
+- [js-image-generator](https://www.npmjs.com/package/js-image-generator) to generate fake images for tests.
+
+## Running tests
+
+Before running tests, you should run `npm install`, to install all dependencies and to download browsers.
```

Comment:
```suggestion
Before running tests, you should run `npm install`, to install all dependencies.
```

Reply:
It's downloading browsers if playwright is in the dependencies, and that represents a huge file... this warning wasn't too much I think.

Reply:
You're right @SimonGrn .

Reply:
As usual, what else is new 😎